.github/workflows/releases.yml

```diff
@@ -20,13 +20,23 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.19
+      - name: Install chocolatey
+        run: |
+          mkdir -p /opt/chocolatey
+          wget -q -O - "https://github.com/chocolatey/choco/releases/download/${CHOCOLATEY_VERSION}/chocolatey.v${CHOCOLATEY_VERSION}.tar.gz" | tar -xz -C "/opt/chocolatey"
+          echo '#!/bin/bash' >> /usr/local/bin/choco
+          echo 'mono /opt/chocolatey/choco.exe $@' >> /usr/local/bin/choco
+          chmod +x /usr/local/bin/choco
+        env:
+          CHOCOLATEY_VERSION: 1.2.0
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2.8.0
         with:
           version: v0.184.0
           args: release
         env:
           GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
+          CHOCOLATEY_API_KEY: ${{ secrets.CHOCOLATEY_API_KEY }}
```

Comment:
An account should be created on chocolatey (could be a maintainers account @clemfromspace) and the API key should be added to github secrets.

Reply:
Added the key / env!